common/src/main/java/com/linecorp/centraldogma/common/PathPattern.java

```diff
@@ -50,6 +59,17 @@ static PathPattern of(String... patterns) {
         return of(ImmutableSet.copyOf(requireNonNull(patterns, "patterns")));
     }
 
+    /**
+     * Creates a path pattern with the {@code pathPatterns}.
+     */
+    static PathPattern of(PathPattern... pathPatterns) {
+        checkArgument(pathPatterns.length > 0, "pathPatterns");
+        if (pathPatterns.length == 1) {
+            return pathPatterns[0];
+        }
+        return new DefaultPathPattern(ImmutableList.copyOf(pathPatterns));
```

Comment:
```suggestion
        return new DefaultPathPattern(ImmutableSet.copyOf(pathPatterns));
```

Reply:
Previously, a new constructor for `DefaultPathPattern(List<PathPattern> verifiedPatterns)` was introduced that would skip the validatePathPattern step. 
I changed the argument type from `List<PathPattern>` -> varargs `PathPattern...`.